src/TestingComponenets/LogIn.test.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable func-names */
+/* eslint-disable react/jsx-no-bind */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from "react-router-dom";
+import userEvent from '@testing-library/user-event';
+import CryptoContext from "../CryptoContext";
+import Login from '../components/Authentication/Login';
+
+test('includes an text fields with text "Enter-Email"', () => {
+    render(
+        <CryptoContext>
+            <BrowserRouter>
+                <Login handleClose={function (): void {
+                    throw new Error('Function not implemented.');
+                }} />
+            </BrowserRouter>
+        </CryptoContext>
+    );
+
+    const PasswordElement = screen.getByTestId('Enter-Email')
+    expect(PasswordElement).toBeInTheDocument();
+
+});
+test('should render a button', () => {
+    render(
+        <CryptoContext>
+            <BrowserRouter>
+                <Login handleClose={function (): void {
+                    throw new Error('Function not implemented.');
+                }} />
+            </BrowserRouter>
+        </CryptoContext>
+    );
+
+    const Loginbutton = screen.getByRole('button', { name: /LOGIN/i });
+    expect(Loginbutton).toBeInTheDocument();
+});
+test('should be able to type in password in the text feild', () => {
+    render(
+        <CryptoContext>
+            <BrowserRouter>
+                <Login handleClose={function (): void {
+                    throw new Error('Function not implemented.');
+                }} />
+            </BrowserRouter>
+        </CryptoContext>
+    );
+    const PasswordElement = screen.getByTestId(/Enter-Password/i)
+    userEvent.type(PasswordElement, "New tittle");
+    expect(PasswordElement).toBeInTheDocument();
+});
+test('should be able to type in email address in the text feild', () => {
```

Comment:
Technically this test only verifies that you can type some sort of text in, not that you can type in an email address.